.github/workflows/tests.yml

```diff
@@ -65,10 +65,21 @@ jobs:
       - name: Run unit tests
         run: go test -v -race -covermode=atomic -coverprofile=coverage.txt -coverpkg=./... ./...
 
-      # disabled as 'pinging codecov' hangs
-      # see https://github.com/codecov/feedback/issues/354
-      # - name: Send coverage to codecov.io
-      #   run: bash <(curl -s https://codecov.io/bash)
+      - name: Send coverage to codecov.io
+        run: bash <(curl -s https://codecov.io/bash)
+
+  codecov:
+    steps:
+    - uses: actions/checkout@main
+    - uses: codecov/codecov-action@v4
+      with:
+        fail_ci_if_error: true # optional (default = false)
```

Comment:
I'd keep this false.

Reply:
I am against all code coverage in general, in my mind this is false metric.

However, if other folks are okay I am not against having it just for informative purposes. Definitely should not be a failure when coverage decreases and this kind of stuff.